com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -55,6 +55,8 @@
 
 	public static final String TEMPLATES_VERSION = "1+";
 
+	private static File zipFile = null;
```

Comment:
Its ok if that is calculated everytime, or are you seeing this get calculated multiple times in a single call?